fitbenchmarking/fitting_benchmarking.py

```diff
@@ -170,6 +169,7 @@ def do_fitting_benchmark_one_problem(prob, minimizers, use_errors=True, count=0,
 
             print("*** with minimizer {0}, Status: {1}, chi2: {2}".format(minimizer_name, status, chi2))
 
+            best_fit = None
```

Comment:
To further improve code readability. Could you add a comment to explain when new variables are first introduced to the code (expect where it is very clear what a variable). Please also don't hesitate to add such where they are currently missing

Reply:
Yes, I was going to create another branch where I would add comments and do a general clean up of the code as soon as I finished with the unit tests.

Reply:
Yes, I was going to create another branch where I add comments and do a general clean up of the code as soon as I finished with the unit tests.

Reply:
OK